client/src/screens/availability/Index.js

```diff
@@ -80,6 +80,7 @@ class Index extends Component {
           keyExtractor={item => item.id}
           renderItem={this.renderItem}
           refreshing={this.props.networkStatus === 4}
+          onRefresh={() => this.props.refetch()}
```

Comment:
I think this is best done with a bind in init but I won't block you

Reply:
actually you might not even need to bind it now that I think about it:
```
onRefresh={this.props.refetch}
```
^ that might be enough because the prop provider should be responsible for binding